codegen/smithy-aws-swift-codegen/src/main/kotlin/software/amazon/smithy/aws/swift/codegen/plugins/DefaultAWSClientPlugin.kt

```diff
@@ -5,24 +5,18 @@
 
 package software.amazon.smithy.swift.codegen.integration.plugins
 
-import software.amazon.smithy.aws.swift.codegen.AWSSwiftDependency
+import software.amazon.smithy.aws.swift.codegen.swiftmodules.AWSClientRuntimeTypes
 import software.amazon.smithy.codegen.core.Symbol
 import software.amazon.smithy.swift.codegen.SwiftWriter
 import software.amazon.smithy.swift.codegen.integration.Plugin
-import software.amazon.smithy.swift.codegen.model.buildSymbol
 
 class DefaultAWSClientPlugin : Plugin {
-    override val className: Symbol
-        get() = buildSymbol {
-            this.name = "DefaultAWSClientPlugin"
-            this.namespace = AWSSwiftDependency.AWS_CLIENT_RUNTIME.target
-            dependency(AWSSwiftDependency.AWS_CLIENT_RUNTIME)
-        }
+    override val className: Symbol = AWSClientRuntimeTypes.Core.DefaultAWSClientPlugin
```

Comment:
Symbols like this are taken from the "Types" file rather than constructed inline, so we can ensure they are correctly configured.